appveyor.yml

```diff
@@ -39,14 +37,14 @@ test_script:
   - yarn test
 #  - yarn test-e2e
   - yarn package-dev
-  - 7z a development.zip release
+#  - 7z a development.zip release
```

Comment:
Does this mean we can also remove the 7zip dependencies, which create those annoying warnings for the dependencies with different os architectures?